chain/types/tipset.go

```diff
@@ -171,9 +170,7 @@ func (ts *TipSet) Equals(ots *TipSet) bool {
 }
 
 func (t *Ticket) Less(o *Ticket) bool {
-	tDigest := blake2b.Sum256(t.VRFProof)
-	oDigest := blake2b.Sum256(o.VRFProof)
```

Comment:
This change was made in https://github.com/filecoin-project/lotus/pull/1295/files#diff-e56411f766fcf002c54b252e1cd56680R176, and it looks rather intentional

Reply:
I'll change go-filecoin to match the prior Lotus behaviour here (but keep using CID bytes to break ties)